### DISCUSSION TITLE
How to have two characters at once?

### DISCUSSION THREAD

**Discussion by UselessAccountThatIMadeTodaymaybe** [Leather128/LeathersFunkinEngine#147] [Q&A] [answered]
How do I have two characters at once like how shaggy and matt does?

**Comment by SrtHero278** [comment, marked as answer]
Make another character and in the config file, use this.
![image](https://user-images.githubusercontent.com/93018286/150037555-f7f4d231-8175-47f2-8332-7c323b4d7933.png)

**Reply by MemTheBun** [reply]
**Omg This can be found in the bside collection of ronezjk**
